Week3/homework/step2-1.js

```diff
@@ -2,8 +2,7 @@
 
 function foo(func) {
   // What to do here?
-  // Replace this comment and the next line with your code
-  console.log(func);
+  return func();
```

Comment:
Nice. But no need to use `return` here, seeing as you don't do anything with the returned value when you call `foo`

Reply:
Dear Nick, thank you for your great feedback. I am gonna implement your feedback ASAP. Good luck.